### DISCUSSION TITLE
RiMusic Pro is it legit or fake?

### DISCUSSION THREAD

**Discussion by Patronza** [fast4x/RiMusic#3998] [General]
Hey all,

I came across this claiming to be RiMusic and that they have a desktop app: https://rimusic.pro/
The website seems to be a bit scarse on some features etc, and no mention of github so i'm assuming that it a fake pretending to be the real RiMusic?

**Comment by fergassss1** [comment]
its surely fake, it would need the updated logo to be legit ig
but it might be a old and left website

**Comment by fast4x** [comment]
It's a fake

**Comment by fast4x** [comment]
> Hey all,
> 
> I came across this claiming to be RiMusic and that they have a desktop app: https://rimusic.pro/
> The website seems to be a bit scarse on some features etc, and no mention of github so i'm assuming that it a fake pretending to be the real RiMusic?

thanks for reporting

**Reply by Patronza** [reply]
No worries. Whilst it wouldnt get me to download from their site as I know the legit updates come here on Github, A newbie to RiMusic would probably fall victim
